BusBoard.Web/ViewModels/BusInfo.cs

```diff
@@ -1,13 +1,42 @@
-﻿namespace BusBoard.Web.ViewModels
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using BusBoard.Api.Postcode;
+using BusBoard.Api.Tfl;
+using BusBoard.Api.Tfl.Responses;
+
+namespace BusBoard.Web.ViewModels
 {
   public class BusInfo
   {
+    public IEnumerable<StopPoint> StopPoints { get; } = new List<StopPoint>();
+    public string ErrorMessage { get; } = "";
+
+    private TflApi tflApi = new TflApi();
+    private PostcodeApi postcodeApi = new PostcodeApi();
+
     public BusInfo(string postCode)
     {
-      PostCode = postCode;
+      try
+      {
+        var postcodeData = postcodeApi.GetPostcodeData(postCode);
+
+        StopPoints = tflApi.GetStopPointsAtLocation(postcodeData.latitude, postcodeData.longitude);
```

Comment:
I would just store two stop points here. It reduces the amount of logic in the cshtml file 🙂

Reply:
Good idea, changed in 0fabf5e 🙂